packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.module.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+
+import component from './scale.component';
+import routing from './scale.routing';
+import autoscaling from '../../../components/autoscaling';
+
+const moduleName = 'ovhManagerPciProjectKubernetesNodePoolsScale';
+angular
+  .module(moduleName, [autoscaling])
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */)
```

Comment:
no lazy loading the module?

Reply:
It was done into:
`packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/index.js`